index.html

```diff
@@ -166,4 +166,30 @@ <h1>About <br> Myself</h1>
     </div>
   </section>
 </body>
-</html>
\ No newline at end of file
+
+<footer class="formsection">
+  <h1>Contact me</h1>
```

Comment:
- To improve you improve the SEO of your site, it is not advisable to have more than one `h1` tag on your HTML page, please fix this 🙏 🙏 . Optionally, you can apply this advice to other sections of your page where you have used `h1` tag 😄 
- For semantic reasons, I advise that you do not wrap this section in a `footer` tag, I suggest you use a `section` tag instead. 🙏